resources/views/welcome.blade.php

```diff
@@ -2,9 +2,16 @@
 
 @section('content')
 <header class="bg-header">
+
+<!-- alert for wrong zip code input -->
+<div class="alert alert-warning alert-dismissible collapse" role="alert" id="zip-alert">
+  <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+  <strong><p>We're sorry, the zip code you entered was not recognized as a valid zip code.</p></strong><p>Please enter a valid zip code and resubmit the form <a href="#search">below.</a> Thank you. </p>
```

Comment:
Since error messages other than zip code related ones are possible in the future, we should consider future proofing this section of code.  One way to do this is to change the hardcoded  "We're sorry, the zip code you entered was not recognized as a valid zip code" to a dynamic mesage.  

One way to do this is to use the $errors variable which holds premade error messages.
`<ul id="errors">`
`@foreach ($errors->all() as $error)`
               ` <li>{{$error}}</li>`
            `@endforeach`
        `</ul>`
(This code would go inside the errors module)

If we impliment the changes above, then the line "Please enter a valid zip code and resubmit the form" should be changed to a message which prompts the user to correct the dynamic error message.  One example of this can be 'Please correct form fields and resubmit'.